shared/task.rs

```diff
@@ -55,6 +62,7 @@ impl Task {
             id: 0,
             command: task.command.clone(),
             path: task.path.clone(),
+            enqueue_at: task.enqueue_at,
```

Comment:
The `from_task` function is used when restarting an existing task.
Copying task.enqueue_at doesn't make much sense in here. Just use `None`

Reply:
Updated.